controllers/payment.controller.js

```
@@ -0,0 +1,110 @@
+        totalPrice: parseFloat(metadata.totalPrice),
+        isSuccess: true,
+        paymentId: event.data.object.id,
+      };
+
+      await handleOrderCreation({ body: formattedData }, res);
+
+      break;
+
+    //Event when payment failed due to card problem or insufficient funds
+    case "payment_intent.payment_failed":
+      console.log("Payment failed");
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)